src/main/scala/com/ing/wbaa/airlock/proxy/api/ProxyService.scala

```diff
@@ -78,19 +101,24 @@ trait ProxyService {
   private def processRequestForValidUser(httpRequest: HttpRequest, s3Request: S3Request, userSTS: User)(implicit id: RequestId) = {
     if (isUserAuthenticated(httpRequest, userSTS.secretKey)) {
       logger.debug(s"Request authenticated: $httpRequest")
-
       if (isUserAuthorizedForRequest(s3Request, userSTS)) {
-        logger.info(s"User (${userSTS.userName}) successfully authorized for request: $s3Request")
-
-        processAuthorizedRequest(httpRequest, s3Request, userSTS)
-
+        // if request is multidelete post
+        val rawQueryString = httpRequest.uri.rawQueryString.getOrElse("")
+        if (httpRequest.entity.contentType.mediaType == MediaTypes.`application/xml` && httpRequest.method == HttpMethods.POST && rawQueryString == "delete") {
```

Comment:
to long `if`
instead of comment maybe:
```scala
val isMultideletePost = httpRequest.entity.contentType.mediaType == MediaTypes.`application/xml` && ....
```